src/lib/prompts.js

```diff
@@ -54,6 +59,17 @@ const prompts = {
         return this.focused.value;
       },
     },
+    {
```

Comment:
Thank you for this!
Please sanitise the returned custom `networkId` value by calling `sanitizeAliasName()` (maybe rename the method as well).

Reply:
Looks good! Thank you. LEt's not force the lower-case then.